justfile

```diff
@@ -34,6 +34,12 @@ clean:
     mix deps.clean --all
     rm -rf .hex .mix .cache lib/mix
 
+boilerplate-update:
+    mkdir -p .bonfire-extension-boilerplate
+    git clone https://github.com/bonfire-networks/bonfire-extension-boilerplate.git --branch main --single-branch .bonfire-extension-boilerplate
+    cd .bonfire-extension-boilerplate && cp -r .github lib .envrc justfile ../ && git -C .. add .envrc .github/workflows/main.yml justfile lib/mix/mess.ex && export COMMIT="$(git rev-parse --short HEAD)" && git -C .. commit -m "Update with https://github.com/bonfire-networks/bonfire-extension-boilerplate commit ${COMMIT}"
```

Comment:
I shorten the last line to:
`cd .bonfire-extension-boilerplate && cp -r .github .envrc .github/workflows/main.yml justfile mess.ex`

Reply:
(note `mess.exs` rather than `mess.ex` and also it doesn't need to be in `lib/` since it gets compiled manually into `mix.exs`)

Reply:
Thank you I should have addressed this